assets/dev/js/editor/regions/navigator/element.js

```diff
@@ -95,8 +95,23 @@ export default class extends Marionette.CompositeView {
 
 		this.childViewContainer = '.elementor-navigator__elements';
 
-		this.listenTo( this.model, 'change', this.onModelChange )
-			.listenTo( this.model.get( 'settings' ), 'change', this.onModelSettingsChange );
+		/**
+		 * Since the way elements are rendered.
+		 * On creating an element on an existing column, everything will work because the JS engine call stack is empty
+		 * and it will create the widget container, but if an element was dragged it will be 2 ticks.
+		 * 1. Create section & column.
+		 * 2. Create the widget in the column.
+		 * For this situation 'linkContainer' without `setTimeout` will fail because the widget container does not exist yet.
+		 */
+		setTimeout( this.linkContainer.bind( this ) );
+	}
+
+	linkContainer() {
```

Comment:
link container to internal state instead of interrupting generic container, also `container.panel` should be removed.